custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssueNavigator.kt

```diff
@@ -2,9 +2,7 @@ package jces1209.vu.page
 
 import org.openqa.selenium.By
 import org.openqa.selenium.WebDriver
-import org.openqa.selenium.support.ui.ExpectedConditions.and
-import org.openqa.selenium.support.ui.ExpectedConditions.or
-import org.openqa.selenium.support.ui.ExpectedConditions.presenceOfElementLocated
+import org.openqa.selenium.support.ui.ExpectedConditions.*
```

Comment:
There's a lot of these `import` diffs in unrelated files.
It makes the PR harder to read, the Git history harder to read and increases risk of Git conflicts.
Also, if it is generated by your IDE, but not by IDE of other collaborators, it risks back-and-forth changes in every PR. I'd align IDE settings within the team and open a dedicated PR to fix the codestyle in the entire repo at once.

Tools like EditorConfig or ktlint can make this alignment easier.

- [ ] less noise in the Git history